pages/establishment/dashboard/views/index.jsx

```diff
@@ -24,6 +26,7 @@ const DashboardLink = ({ path }) => (
 );
 
 const Index = ({
+  establishment,
   establishment: {
```

Comment:
you have 2 keys referring to the same thing here, I'd either pick off the establishment properties you need for the banner, or get rid of the destructuring and use `establishment.property` wherever we need it